common/lib/xmodule/xmodule/js/src/video/09_video_caption.js

```diff
@@ -747,6 +737,21 @@
                     self.isMouseFocus = false;
                     self.rendered = true;
                     self.state.el.addClass('is-captions-rendered');
+
```

Comment:
This "attr" value also needs to be escaped (because you don't know what may be in the translation). @robrap is the best practice here just to use _.escape?